proto/tendermint/abci/types.proto

```diff
@@ -285,6 +349,21 @@ message EventAttribute {
   bool   index = 3;  // nondeterministic
 }
 
+// ExecTxResult contains results of executing one individual transaction.
+//
+// * Its structure is equivalent to #ResponseDeliverTx which will be deprecated/deleted
+message ExecTxResult {
+  uint32         code       = 1;
+  bytes          data       = 2;
+  string         log        = 3;  // nondeterministic
+  string         info       = 4;  // nondeterministic
+  int64          gas_wanted = 5 [json_name = "gas_wanted"];
```

Comment:
I think this is the default JSON name anyway, unless you explicitly request mixed case.

Reply:
Yep, I'll remove the JSON annotation here. I will also go through the whole PR again to see if this comment also applies elsewhere